pkg/core/data/ipc/transactions/transaction.go

```diff
@@ -190,7 +190,7 @@ func (t Transaction) Obfuscated() bool {
 // Values returns the amount and fee spent in this transaction.
 // TODO: implement.
```

Comment:
remove the TODO ;P

Reply:
It's still not entirely implemented, that's why I left it 😅